cluster/gce/util.sh

```diff
@@ -331,6 +331,27 @@ function wait-for-jobs {
   fi
 }
 
+# Robustly try to create a static ip.
+# $1: The name of the ip to create
+function create-static-ip {
+  detect-project
+  local attempt=0
+  while true; do
+    if ! gcloud compute addresses create "$1" \
```

Comment:
this is missing a `fi`.

`/jenkins-master-data/jobs/kubernetes-pull-build-test-e2e-gce/workspace@7/hack/e2e-iternal/../../cluster/gce/util.sh: line 352: syntax error near unexpected token `done'`